scripts/register_saft_addresses.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Register SAFT accounts script
+ *
+ * ts-node scripts/register_saft_addresses.ts "network" "path to csv with addresses"
+ */
+
+import fs from 'fs'
+import { utils, constants, providers, Wallet } from 'ethers'
+import { TimeLockRegistry } from '../build/types/TimeLockRegistry'
+import { TimeLockRegistryFactory } from '../build/types/TimeLockRegistryFactory'
+import { TrustToken } from '../build/types/TrustToken'
+import { TrustTokenFactory } from '../build/types/TrustTokenFactory'
+
+export const txnArgs = { gasLimit: 2_000_000, gasPrice: 20_000_000_000 }
+
+export interface SaftAccount {
+  address: string,
+  amount: string,
+}
+
+const toTrustToken = (amount: string) => utils.parseUnits(amount, 8)
+const sum = (numbers: utils.BigNumber[]) => numbers.reduce((cumSum, value) => cumSum.add(value), constants.Zero)
```

Comment:
`cumSum` doesn't sound fortunate. Maybe simply sth like this?
```suggestion
const sum = (numbers: utils.BigNumber[]) => numbers.reduce((a, b) => a.add(b), constants.Zero)
```

Reply:
cumsum is widely used https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/cumsum, https://numpy.org/doc/stable/reference/generated/numpy.cumsum.html

the word might be funny though

Reply:
Still `(a, b) => a.add(b)` imo looks better.

Reply:
I never remember which argument is which in reduce. But in this case the order doesn't matter, so it's ok I guess